src/Stratis.Features.FederatedPeg/NetworkHelpers/IPEndPointComparer.cs

```diff
@@ -1,21 +1,22 @@
 ﻿using System.Collections.Generic;
 using System.Net;
+using Stratis.Bitcoin.Utilities.Extensions;
```

Comment:
Can we move this whole class to `Stratis.Bitcoin.P2P`? I would like to make use of this possibly when retrieving peers from the address manager :)

Reply:
Since we have the `public static bool Match(this IPEndPoint endPoint, IPEndPoint matchWith)` method in `Stratis,Bitcoin.Utilities.Extensions.IPExtensions` perhaps we should move `IPEndPointComparer` to `Stratis.Bitcoin.Utilities`  to make it more broadly usable?

Reply:
For peer matching also consider using `public bool HasRemoteIPAddress(IPAddress ip, int? port = null)`.

Reply:
Moved to `Stratis.Bitcoin.Utilities`.